rpnpy/calculator.py

```diff
@@ -546,6 +547,7 @@ def _tryEvalExec(self, command, modifiers, count):
                 value = EngNumber(command)
             except decimal.InvalidOperation:
                 try:
+                    variables_before = set(self._variables.keys())
```

Comment:
You don't need the `.keys()` here. The `dict` is being iterated, so you get the keys as a result.

Also, could you use camelCase? I know it's a tiny/silly issue, but I'd prefer to keep the code consistent in that respect. Sorry!